docs/reference/commands/enable.rst

```diff
@@ -0,0 +1,111 @@
+==========
+``enable``
+==========
+
+Enables a feature or service on the cluster.
+
+Usage:
+
+.. code-block:: none
+
+   microceph enable [flags]
+   microceph enable [command]
+
+Available commands:
+
+.. code-block:: none
+
+   mds         Enable the MDS service on the --target server (default: this server)
+   mgr         Enable the MGR service on the --target server (default: this server)
+   mon         Enable the MON service on the --target server (default: this server)
+   rgw         Enable the RGW service on the --target server (default: this server)
+
+Global flags:
+
+.. code-block:: none
+
+   -d, --debug       Show all debug messages
+   -h, --help        Print help
+       --state-dir   Path to store state information
+   -v, --verbose     Show all information messages
+       --version     Print version number
+
+``mds``
+-------
+
+Enable the MDS service on the --target server (default: this server)
```

Comment:
s/this server/localhost ?

Reply:
I'm somewhat partial to `this server` because `localhost` could be misconstrued to mean "listen only on the localhost device i.e. 127.0.0.1" which would be incorrect.